lib/internal/fs/promises.js

```diff
@@ -31,6 +31,8 @@ const {
     ERR_INVALID_ARG_TYPE,
     ERR_INVALID_ARG_VALUE,
     ERR_METHOD_NOT_IMPLEMENTED,
+    ERR_STREAM_PREMATURE_CLOSE,
+    ERR_OUT_OF_RANGE
```

Comment:
nit: ASCII order, trailing comma.
```suggestion
    ERR_OUT_OF_RANGE,
    ERR_STREAM_PREMATURE_CLOSE,
```